oauth2-openapi/README.md

```diff
@@ -3,6 +3,56 @@
 Protect your API against unauthorized access **without changing your application**. 
 Our newest Gate plugin automatically enforces OpenAPI security checks, so you can implement fine-grained access control for your APIs and workloads without writing any extra code.
 
+## Quick Start
+The guide below provides context and a deeper-dive on how to setup OAuth 2.0 Client Credentials as well enforcing OAuth 2.0 scopes on an OpenAPI spec in Gate. 
+To see this in action, you can follow these quick steps: 
+
+1. Create a free account at https://console.slashid.dev/signup 
+2. Navigate to https://console.slashid.dev/configuration/general to note down the Organization ID and to generate an API Key for SlashID
+2. Run `docker-compose pull` to get the latest images
+3. Make sure you have `curl` and `jq` installed and create a client ID/secret pair with SlashID using `./create_client.sh <Organization ID> <API Key>`
+4. Set the client ID and secret in `gate.yaml`
+5. Use the client ID/secret to calculate the basic authorization header with `./request_token.sh <client_id> <client_secret>`. Note down the access token, as it will be used as the bearer for future requests. This script requests a token with `customers:read` and `customers:create` scopes
+6. Run `docker-compose up` to start Gate and the backend server
+
+You can now make requests to `http://localhost:5000`.
+
+**Note that for this example we are using an echo server, this means it does not implement the API described above, but is sufficient to demonstrate that the security schemes are being enforced.**
+
+Let's see it in action: 
+
+```bash
+curl -X POST http://localhost:5000/customers -H "Authorization: Bearer <access_token>" --data '{ "customer_name": "test_customer", "customer_tax_id": "4"}'
+< HTTP/1.1 200 OK
+< Content-Length: 605
+< Content-Type: text/plain
+< Date: Tue, 14 Nov 2023 12:24:28 GMT
+< Via: 1.0 gate
+< 
+...
+{ "customer_name": "test_customer", "customer_tax_id": "4"}
+
+```
+We receive a 200 because we are authorized to create the user, let's try to delete it now
```

Comment:
```suggestion
We receive a 200 because we are authorized to create a customer - the access token has the correct scopes for the operation. Now let's try to delete a customer:
```